src/Table/Table.style.js

```diff
@@ -0,0 +1,7 @@
+export default {
+  display: 'grid',
+  gridAutoFlow: 'row',
+  backgroudColor: 'white',
+  borderRadius: '2px',
+  boxShadow: 'etvasCard'
```

Comment:
This shouldn't be part of the `Table` component. If the user wants it to look like a card something like this is possible:
```jsx
<Card>
  <Table ... >
   </Table>
</Card>
```

The `Table` component should only arrange children in a table.